brevbaker/src/main/kotlin/no/nav/brev/brevbaker/PDFByggerService.kt

```diff
@@ -3,5 +3,7 @@ package no.nav.brev.brevbaker
 import no.nav.pensjon.brev.PDFRequest
 
 interface PDFByggerService {
-    suspend fun producePDF(pdfRequest: PDFRequest): PDFCompilationOutput
-}
\ No newline at end of file
+    suspend fun producePDF(pdfRequest: PDFRequest, url: String = PATH): PDFCompilationOutput
```

Comment:
Bærer ikke url konnotasjonen at det er snakk om potensielt hele `<protokoll>://<host>:<port>/<path>`?

Reply:
Jo, var det som slo meg da eg endra til path for konstanten - men gløymte heilt å vera konsekvent 🙈